src/test/ui/consts/miri_unleashed/drop.rs

```diff
@@ -0,0 +1,20 @@
+// compile-flags: -Zunleash-the-miri-inside-of-you
+// ignore-x86 FIXME: missing sysroot spans (#53081)
+#![deny(const_err)]
+
+use std::mem::ManuallyDrop;
+
+fn main() {}
+
+static TEST_OK: () = {
+    let v: Vec<i32> = Vec::new();
+    let _v = ManuallyDrop::new(v);
+};
+
+// Make sure we catch executing bad drop functions.
+// The actual error is located in `real_drop_in_place` so we can't capture it with the
```

Comment:
you can use `error-pattern` on the file or whatever that comment tag is called.

But in general I think it would be useful to have `//~ LABEL` annotations that just check for any sub-span on an error

Reply:
> But in general I think it would be useful to have //~ LABEL annotations that just check for any sub-span on an error

That would be a new compiletest feature, right, nothing I can do now?

Reply:
yea it was just random musings